### ISSUE TITLE
Suggestion: Add robust information UI

### ISSUE THREAD

**Issue by Ramdat** [cameroncondry/cbc-kitten-scientists#144]
KS has a lot of great automation, but it lacks in informative features.

Integrate popular calculators into the game itself. If it is hooked into the game, it can automatically calculate the information without manually inputting them into a spreadsheet and can display the information in-game instead of in a third-party interface. Although ambitious, it would greatly improve the player's ability to make informed decisions and make KS the go-to script.

Calculators include:
* Farmers with catnip refinement vs Woodcutters using all existing upgrades and other modifiers.
* Blueprint efficiency for Magneto/Steelworks/Reactor.
* Steel efficiency for Magneto/Steelworks (power and production), Mansion/Space Station, Oil Well/Biolab.
* Eludium efficiency for Moonbase and Cryostation.
* Unicorn calculator for unicorn related buildings.

These can be found [here](https://www.reddit.com/r/kittensgame/comments/3ymz7x/every_google_calculator_i_could_find/).

Additional information:
Display the food tick during all seasons to determine if kittens can survive the winter in the early game. Include cold winter for worst-case scenario.

**Comment by oliversalzburg** [comment]
This sounds interesting. I've never personally used any of those spreadsheets, but I can see them as very beneficial in the early and mid-game.

Sounds like a lot of work to implement though.

**Comment by Ramdat** [comment]
@oliversalzburg Indeed, I understand that it's quite ambitious. Although, the added features would improve the script rather significantly for newer players. 

Is it too much work for you to pursue?

**Comment by oliversalzburg** [comment]
@Ramdat Well, I have a couple more changes planned myself. But everyone is welcome to contribute :)

Either way, there were open feature requests on this tracker for quite some time (like support for Space). At some point someone might implement it. But you might be so far into the late-game by then, that you don't need the feature anymore ;D
